### ISSUE TITLE
altdoc 0.4.0

### ISSUE THREAD

**Issue by etiennebacher** [etiennebacher/altdoc#286]
0.3.0 was released about 5 months ago, in February. There are several fixes and new features that users could benefit from (e.g. #285). 

@vincentarelbundock do you plan to do some changes in the next few weeks? I'm not in a rush to release 0.4.0, but if we know nothing new is going to come before some time I can release it.

**Comment by vincentarelbundock** [comment]
Yeah, I agree that we're due for a release. After https://github.com/etiennebacher/altdoc/pull/287 is merged, I don't plan to start any big work in the near future.
